Tests/SonarQube.MSBuild.Tasks.IntegrationTests/E2ETests/E2EAnalysisTests.cs

```diff
@@ -356,6 +520,15 @@ private static void AddEmptyContentFile(ProjectDescriptor descriptor, string pro
             descriptor.ContentFiles.Add(emptyFilePath);
         }
 
+        private static string CreateEmptyFile(string folder, string extension)
+        {
+            string emptyFilePath = Path.Combine(folder, "empty_" + Guid.NewGuid().ToString() + ".xxx");
+            emptyFilePath = Path.ChangeExtension(emptyFilePath, extension);
```

Comment:
Why don't you create the file with the given extension directly and why do you need to write an empty string to it?